lms/static/js/student_account/views/LoginView.js

```diff
@@ -212,6 +212,20 @@
                     msg = gettext('An error has occurred. Check your Internet connection and try again.');
                 } else if (error.status === 500) {
                     msg = gettext('An error has occurred. Try refreshing the page, or check your Internet connection.'); // eslint-disable-line max-len
+                } else if (error.responseJSON !== undefined && error.responseJSON.error_code === 'inactive-user') {
+                    msg = StringUtils.interpolate(
+                    // xss-lint: disable=javascript-concat-html
+                    gettext('In order to sign in, you need to activate your account.<br/><br/>' +
+                            'We just sent an activation link to <strong> %(email)s </strong>. If ' +
+                            // xss-lint: disable=javascript-concat-html
+                            ' you do not receive an email, check your spam folders or ' +
+                            ' <a href=%(support_url)s>contact %(platform_name)s Support</a>.'),
+                        {
+                            email: error.responseJSON.email,
+                            platform_name: this.platform_name,
+                            support_url: 'https://support.edx.org/'
+                        }
+                );
```

Comment:
This is not the correct way to interpolate HTML into text, see this for an example https://github.com/edx/edx-platform/blob/2383fb3fa6f43f048b3df9d1007499afa06a5658/lms/static/js/student_account/views/LoginView.js#L164-L183